_includes/configuration/images-upload-handler.md

```diff
@@ -6,7 +6,8 @@ The upload handler function takes four arguments:
 
 * `blobInfo`
 * A `success` callback
-* A `failure` callback
+* A `failure` callback that takes an error-message and an optional object containing:
+    * `remove` Removes the image from the document, defaults to `false`
```

Comment:
```suggestion
    * `remove` - Removes the image from the document, defaults to `false`
```

Reply:
Or however @tylerkelly13 does these things; should maybe be a `:` instead?

Reply:
either's fine :)